addons/controllers/clusterbootstrap_controller.go

```diff
@@ -364,10 +364,13 @@ func (r *ClusterBootstrapReconciler) createOrPatchClusterBootstrapFromTemplate(c
 	clusterBootstrapTemplate := &runtanzuv1alpha3.ClusterBootstrapTemplate{}
 	key := client.ObjectKey{Namespace: r.Config.SystemNamespace, Name: tkrName}
 	if err := r.Client.Get(r.context, key, clusterBootstrapTemplate); err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Info("no ClusterBootstrapTemplate associated with cluster")
```

Comment:
**suggestion**: Can we log cluster name and namespace.

Reply:
I believe they are logged on already, but, I will check.

Reply:
Ah ok. Good then

Reply:
Yes, the cluster name and namespace are added to the logs here: https://github.com/vmware-tanzu/tanzu-framework/blob/50c237ec19e948f4999679ec44d5eb6b8ca407dd/addons/controllers/clusterbootstrap_controller.go#L141

Reply:
I think this is an error. If there is a classy cluster it must have a CBT unless we want to filter out TKGS legacy-tkr which we should do explicitly instead of ignoring all errors.